app/controllers/api/v2/site_statuses_controller.rb

```diff
@@ -7,11 +7,19 @@ def update
         site_status = authorize SiteStatus.find(params[:id])
         site_status.update site_status_params
 
+        if no_vacancies?
```

Comment:
```suggestion
        if site_status.no_vacancies?
```
Does this work? If so you can delete private method

Reply:
Good spot. I've made the change